dockers/docker-database/docker-database-init.sh

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
 
+# For linux host namespace, in both single and multi ASIC platform use the loopback interface
+# For other namespaces, use eth0 interface which is connected to the docker0 bridge in the host.
+if [[ $NAMESPACE_ID == "" ]]
+then
+    INTFC=lo
+else
+    INTFC=eth0
+    LOOPBACK_IP=127.0.0.1
+fi
+local_ip=$(ip -4 -o addr show $INTFC | awk '{print $4}' | cut -d'/' -f1)
```

Comment:
>$INTFC [](start = 30, length = 6)

multiple ip address on lo ?

Reply:
@qiluo-msft , we don't support configuring "`lo`" anymore. I don't think there would be a case for multiple ip address

Reply:
Thanks @prince.sunny!
However, no harm to add `head -1` to get the first line so we are super safe.

---
In reply to: [441926773](https://github.com/Azure/sonic-buildimage/pull/4779#discussion_r441926773) [](ancestors = 441926773)

Reply:
Updated